spec/menu_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'menu'
+
+describe Menu do
+  let(:item) { double:item }
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.